src/Hintman/Effect/TokenStorage.hs

```diff
@@ -32,7 +32,6 @@ insertTokenImpl
     -> GitHubToken
     -> m ()
 insertTokenImpl owner repo key = do
-    putTextLn $ "Inserting token for: " <> show owner <> "/" <> show repo
```

Comment:
Should not this ti be the logging message?

Reply:
Yes, it should. I wanted to move it to a separate function (because I don't want to have logging in this effect) but forgot...